### PR TITLE
Tao 790 apip authoring button

### PR DESCRIPTION
**Pull request by ssipasseuth**

### CODE REVIEW

File: controller/ApipCreator.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ * @license GPLv2  http://www.opensource.org/licenses/gpl-2.0.php
+ */
+class ApipCreator extends tao_actions_CommonModule
+{
+
+    public function index(){
+
+        $itemUri = tao_helpers_Uri::decode($this->getRequestParameter('id'));
+        if(is_null($itemUri) || empty($itemUri)){
+            throw new tao_models_classes_MissingRequestParameterException("id");
```
Comment (hutnikau) [inline controller/ApipCreator.php]: `\` missed before class name.

**Comment by ssipasseuth** [comment]
oops, the last three commits are on the wrong branch !

**Comment by ssipasseuth** [comment]
Ok, I fixed the missng slash. Please review it again.

**Comment by hutnikau** [comment]
Great, thanks!
Only one small advice. Please note that in php files we are following PSR-2 standard.
http://www.php-fig.org/psr/psr-2/

**Comment by ssipasseuth** [comment]
thks, I shall take care of that.

**Pull request merged**